File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/java/python/p03085.py
Here is the Python version of the given Java code:

```python
helix = input()
complements = {'A': 'T', 'T': 'A', 'C': 'G', 'G': 'C'}
print(complements.get(helix, ""))
```

The above Python code reads from standard input, just like the Java code. It uses a dictionary to map each character in the helix to their complement. The `print` statement outputs the complement, and the `get` method ensures that if the input is not in the dictionary, an empty string is returned (which is equivalent to having no `default` in the Java `switch` statement).

File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/python/java/p03085.java
Here is the equivalent Java code for the given Python program:

```java
import java.util.Scanner;

public class DnaComplement {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String b = scanner.nextLine();
        scanner.close();

        if (b.equals("A")) {
            System.out.println("T");
        } else if (b.equals("C")) {
            System.out.println("G");
        } else if (b.equals("G")) {
            System.out.println("C");
        } else {
            System.out.println("A");
        }
    }
}
```

This Java code achieves the same functionality as your provided Python script. It takes an input from the user, checks for a matching DNA base and outputs its complement. Just like the Python code, it assumes that only one character will be inputted representing a DNA base.
